.github/workflows/main_workflow.yml

```diff
@@ -198,12 +198,15 @@ jobs:
 
           # build required images
           pip install .
-          autonomy deploy build deployment valory/oracle_hardhat deployments/keys/hardhat_keys.json --local
+          autonomy fetch --local --service valory/hello_world
+          
+          cd hello_world
+          autonomy build-images
+          autonomy build-images --dev
+          autonomy build-images --dependencies
+          autonomy build-images --dependencies --version 1.0.0
+          autonomy deploy build ../deployments/keys/hardhat_keys.json
```

Comment:
What does `autonomy deploy build ..` do?

Reply:
@angrybayblade

Reply:
`autonomy deploy build ../deployments/keys/hardhat_keys.json` is the whole command, `autonomy deploy build` builds deployments for the available service.